assets/post_install_macOS.sh

```diff
@@ -28,22 +31,23 @@ else
     APP_DIR="$HOME"/Applications
     PERMS=""
 fi
-SPI_APP_DIR="${APP_DIR}/Scientific-Python"
-logger -p 'install.info' "📓 SPI_APP_DIR=$SPI_APP_DIR"
+PKG_APP_DIR="${APP_DIR}/${INSTALLER_NAME}"
+logger -p 'install.info' "📓 PKG_APP_DIR=$PKG_APP_DIR"
 
-logger -p 'install.info' "ℹ️ Moving root SP .app bundles from $APP_DIR to $SPI_APP_DIR."
-$PERMS mv "$APP_DIR"/Scientific\ Python\ *.app "$SPI_APP_DIR"/
+logger -p 'install.info' "ℹ️ Moving root project .app bundles from $APP_DIR to $PKG_APP_DIR."
+# Set this to match the names generated from the menu package.
+$PERMS mv "${APP_DIR}"/Scientific\ Python\ *.app "$PKG_APP_DIR"/
```

Comment:
Just so I'm clear: it isn't possible to remove `Scientific\ Python` here too?  IIUC this is for all the menu items, and there's no guarantee that they'll always have a nice clean prefix that glob-matches all of them...  but in our case there is (by design).  So in theory we could use a variable here for our installer, but it wouldn't generalize for use cases where the menu items have arbitrary names. Is that right?